src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -17,29 +22,68 @@ class TransactionRecordQuery(Query):
     Represents a query for a transaction record on the Hedera network.
     """
 
-    def __init__(self, transaction_id: Optional[TransactionId] = None):
+    def __init__(
+        self,
+        transaction_id: Optional[TransactionId] = None,
+        include_duplicates: bool = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)